include/ut_kvp.h

```diff
@@ -36,6 +36,7 @@ typedef enum
     UT_KVP_STATUS_NO_DATA,           /**!< No data to process. */
     UT_KVP_STATUS_NULL_PARAM,        /**!< Null parameter passed. */
     UT_KVP_STATUS_INVALID_INSTANCE,  /**!< Invalid KVP instance handle. */
+    UT_KVP_STATUS_INVALID_KEY,       /**!< Invalid Key passed by user. */
```

Comment:
there's no mention of this enum in any of the header file documentation for any api?

Reply:
removed this.